### DISCUSSION TITLE
DIR /a to displays files in specified directory and all subdirectories

### DISCUSSION THREAD

**Discussion by Torinde** [dosbox-staging/dosbox-staging#3972] [Ideas]
I would like `dir string*.* /s/a` to work as on DOS:
- `/s`: search specified directory and all subdirectories
- `/a` (without extra letter specified): search regardless of attributes (e.g. include hidden, etc.)
![image](https://github.com/user-attachments/assets/e2a3fe8e-dda3-4972-ad69-d73074c9618b)

**Comment by johnnovak** [comment]
`/s` might be useful, `/a` seems to be extremely niche.

**Reply by johnnovak** [reply]
You can raise a ticket for `/s` @Torinde if you want. Closing this.
